splunklib/client.py

```diff
@@ -1883,6 +1908,25 @@ def count(self):
         """
         return int(self.content.get('triggered_alert_count', 0))
 
+class DeploymentApp(Entity):
+    pass
+
+
+class DeploymentApps(Collection):
+    """This class represents a collection of deployment apps."""
+    def __init__(self, service):
+        Collection.__init__(self, service, PATH_DEPLOYMENT_APPS, item=ServerClass)
+
+class DeploymentClient(Entity):
+    pass
+
+
+class DeploymentClient(Collection):
+    """This class represents a collection of deployment clients."""
+    def __init__(self, service):
+        Collection.__init__(self, service, PATH_DEPLOYMENT_CLIENTS, item=ServerClass)
```

Comment:
You'll want to change the name of the collection to `DeploymentClients` (plural) and this line to `item= DeploymentClient`